spec/api/v2/groups_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::GroupsController, type: :request do
+  describe 'api' do
+
+    # INITIALIZE USERS - CREATE NEW USER OR GET EXISTING USER
+    # NOTE: THESE USERS SHOULD NOT EXIST
+    # NOTE: DATABASE ENTRIES IN BEFORE :ALL ARE NOT FLUSHED AUTOMATICALLY
+    before :all do
+      @user_1 = create(:user, login: 'user_1') rescue nil
+      @user_1 = @user_1 || User.find_by_login('user_1')
+    end
+
+    after :all do
+      @user_1.delete
+    end
+
+    # GET GROUPS
+    it 'get_groups' do
+      get '/api/v2/groups'
+      json_response = JSON.parse(response.body)
+
+      # JUST CHECK FOR VALID RESPONSE
+      expect(json_response["status"]).to eq(200)
+    end
+
+    # GET GROUP
+    it 'get_group' do
+      get '/api/v2/groups/55'
+      json_response = JSON.parse(response.body)
+
+      group = json_response["data"]
+      expect(group["name"]).to eq("technicians")
```

Comment:
The default database SHOULD have the technician group as group 55.  I'm pretty sure mine does, so I'm not sure why yours does not. In any event, this is super-critical because we are assuming that the Technicians group is created and is not obliterated.  That was one of the original assumptions about the whole sprint before we started it! (and it will be resolved with the "roles & permissions" in v3.

Reply:
now passing